### ISSUE TITLE
What happens when a storage account's secret gets rotated?

### ISSUE THREAD

**Issue by ejschoen** [kubernetes-sigs/blob-csi-driver#1566]
I have a question--but not a support request.  Apologies if indeed the boilerplate I erased was meant to prevent asking questions specific to blob-csi-driver.

What happens if a storage account's secret gets rotated? Does the secret referenced by csi.nodeStateSecretRef need to be updated as well, or will an active fuse driver continue to work? (We could rotate the referenced secret with the secrets-store.csi.x-k8s.io/v1 SecretProviderClass by syncing with the vault from which the storage account's secret originates.)

This is maybe a followup to issue #275.

**Comment by andyzhangx** [comment]
@ejschoen yes, you need to update the secret referenced by csi.nodeStateSecretRef and then restart pod to make volume remount happen.
